layers/!lang/ruby/README.org

```diff
@@ -87,3 +90,14 @@ a couple of useful keybindings:
 |-------------+---------------------|
 | ~SPC m t b~ | run test file       |
 | ~SPC m t t~ | run test at pointer |
+
+** rubocop
+
+| Key binding   | Description                   |
+|---------------+-------------------------------|
+| ~SPC m r a f~ | check current file            |
+| ~SPC m r a d~ | check current directory       |
+| ~SPC m r a p~ | check current project         |
```

Comment:
OK, should I remove these? How do I do it? Just push a new commit, send a new PR or something else?